File: sandbox.client/src/model/PhoneDisplay.ts
export class PhoneDisplay {
  public id: number;
  public type: string;
  public number: string;

  public static create(a: any): PhoneDisplay {
    const ret = new PhoneDisplay();
    ret.assign(a);
    return ret;
  }

  constructor() {
    this.id = 0;
    this.type = "HOME";
    this.number = null;
  }

  assign(a: any) {
    this.id = a.id;
    this.type = a.type;
    this.number = a.number;
  }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/PhoneDisplay.java
package kz.greetgo.sandbox.controller.model;

public class PhoneDisplay {
  public int id;
  public String type;
  public String number;

  public PhoneDisplay() {
    this.id = 0;
    this.type = "HOME";
  }

  public PhoneDisplay(int id, String type, String number) {
    this.id = id;
    this.type = type;
    this.number = number;
  }
}
